jupyter_sphinx/execute.py

```diff
@@ -87,16 +59,16 @@ class JupyterKernel(Directive):
     final_argument_whitespace = False
     has_content = False
 
-    option_spec = {
-        'id': directives.unchanged,
-    }
+    option_spec = {"id": directives.unchanged}
 
     def run(self):
-        return [JupyterKernelNode(
-            '',
-            kernel_name=self.arguments[0].strip() if self.arguments else '',
-            kernel_id=self.options.get('id', '').strip(),
-        )]
+        return [
+            JupyterKernelNode(
+                "",
+                kernel_name=self.arguments[0].strip() if self.arguments else "",
+                kernel_id=self.options.get("id", "").strip(),
+            )
+        ]
 
 
 class JupyterKernelNode(docutils.nodes.Element):
```

Comment:
Shouldn't the nodes be in `ast`?

Reply:
I was torn on this one - on one hand they're clearly AST objects, on the other they are explicitly about executing things. Do you have a preference? I'm happy to go with either

Reply:
I don't have a strong preference neither, but I think like putting all nodes (including this one) would make codebase a bit easier to navigate. Therefore I suggest indeed migrating this to `ast`. It's not a big point, and I'm happy with it either way.

Reply:
done in the latest push!